### DISCUSSION TITLE
Addition of Persmission Grant

### DISCUSSION THREAD

**Discussion by decipher3114** [RikkaApps/Shizuku#182] [Ideas]
Please add inbuilt terminal or add a function to grant WRITE_SECURE_SETTINGS permission directly from app or add this functionality in App-Ops.
Please Sir.

**Comment by TommyPollyblank** [comment]
Ideas

**Comment by RikkaW** [comment]
Shizuku already provides `rish` (see "Use Shizuku in terminal apps" in Shizuku app) which allows you to access the shell from Shizuku in any terminal apps. Also, create a fully functional terminal is not as easy as you thought. There are a lot of terminal apps in the world, it's not necessary to spend energy to create a terminal again.
